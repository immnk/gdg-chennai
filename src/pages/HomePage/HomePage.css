$background: #f4f1bb;
$text-gray-color: #bfacaa;
$text-dark-gray: #474448;
$text-white-color: #fff;
$text-header-color: #330c2f;
$header-font: 'ZCOOL QingKe HuangYou', cursive;
$code-font: 'Roboto Mono', monospace;

.root {
  position: relative;
  height: 100%;
  background-color: $background;
  display: flex;
  flex-direction: column;
  justify-content: center;
  padding: 2.4rem;
  text-align: center;
  font-family: $code-font;
  color: $text-gray-color;
}

.header {
  font-size: 5rem;
  font-family: $header-font;
  letter-spacing: 0.5rem;

  & + p {
    padding: 1.2rem;
    color: $text-dark-gray;
    font-size: 1.3rem;
  }
}

.highlight {
  color: $text-header-color;
  font-family: $header-font;
}

.iconsSocial {
  i {
    color: $text-gray-color;
    font-size: 3rem;
    padding: 1rem;
    &:hover {
      color: $text-dark-gray;
    }
  }
}

/* .devImage {
  position: absolute;
  top: 25%;
  transform: translateX(-50%) translateY(-50%);
  left: 50%;
  width: 100px;
  height: 100px;
  opacity: 0.5;
  background: #f4f1bb url(./dev.png) no-repeat;
  background-size: contain;
  border-radius: 50%;
} */
