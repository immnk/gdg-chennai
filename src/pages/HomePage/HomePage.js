import React, { PureComponent } from 'react';

import css from './HomePage.css';

export default class HomePage extends PureComponent {
  render() {
    return (
      <div className={css.root}>
        Hello home
        <a href="#/more">go</a>
      </div>
    );
  }
}
