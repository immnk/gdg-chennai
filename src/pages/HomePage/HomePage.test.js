import React from 'react';
import { shallow } from 'enzyme';
import HomePage from './HomePage';

const renderHomePage = (props = {}) => {
  const defaultProps = {};

  return <HomePage {...defaultProps} {...props} />;
};

const shallowRenderHomePage = (props) => shallow(renderHomePage(props));

describe('<HomePage>', () => {
  it('Should render', () => {
    const actual = shallowRenderHomePage();
    expect(actual).toMatchSnapshot();
  });
});
