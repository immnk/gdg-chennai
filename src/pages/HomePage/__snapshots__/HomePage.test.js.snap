// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage> Should render 1`] = `
<div
  className="root"
>
  Hello home
  <a
    href="#/more"
  >
    go
  </a>
</div>
`;
